playbooks/roles/airship-deploy-ucp/tasks/main.yml

```diff
@@ -47,38 +53,48 @@
     - install
 
 - name: Create site-definition yaml
-  become: yes
+  delegate_to: localhost
   template:
     src: '{{ playbook_dir }}/../site/soc/site-definition.yaml'
-    dest: '{{ site_path }}'
+    dest: '{{ local_temp_path }}'
   tags:
     - install
     - update_airship_osh_site
 
 - name: Create site software manifests
-  become: yes
+  delegate_to: localhost
   template:
     src: '{{ item.src }}'
-    dest: '{{ site_path }}/software/{{ item.path }}'
+    dest: '{{ local_temp_path }}/software/{{ item.path }}'
   with_filetree: '{{ playbook_dir }}/../site/soc/software/'
   when: item.state == 'file'
   tags:
     - install
     - update_airship_osh_site
 
 - name: Create site secrets manifests
-  become: yes
+  delegate_to: localhost
   template:
     src: '{{ item.src }}'
-    dest: '{{ site_path }}/secrets/{{ item.path }}'
+    dest: '{{ local_temp_path }}/secrets/{{ item.path }}'
   with_filetree: '{{ playbook_dir }}/../site/soc/secrets/'
   when: item.state == 'file'
   tags:
     - install
     - update_airship_osh_site
 
+- name: Sync local site dir with remote
+  synchronize:
+    src: "{{ local_temp_path }}"
+    dest: "{{ site_path }}"
+    recursive: yes
+
+- name: Remove local site
+  file:
+    path: "{{ local_temp_path }}"
+    state: absent
+
 - name: Create pod scale profile
-  become: yes
```

Comment:
Why can you remove `become` here? It does not seem relevant to the other checks

Reply:
good point, I did miss this one

Reply:
fixed!